### PR TITLE
Change SIG Core Banner

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Change SIG Core Banner

### CODE REVIEW

File: content/banner/slides.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 bannerDuration: 5000
 slides:
-  - text: "Join us for the Core SIG meeting, February 27!"
-    url: "https://docs.google.com/document/d/1vWleTon7sJIk0teee4SoVezS4mR3K8TlkvXkgLJwRD8/edit#heading=h.m5wq2qjk6l90"
-    startDate: "21/02/2019"
-    endDate: "27/02/2019"
+  - text: "Join us for the Core SIG meeting, March 13!"
+    url: "https://docs.google.com/document/d/1vWleTon7sJIk0teee4SoVezS4mR3K8TlkvXkgLJwRD8/edit#heading=h.ds9remfq6dch"
```
Comment (pkosiec) [inline content/banner/slides.yml]: Do we need to provide heading anchor? We won't need to modify this URL for future Core SIG meetings

File: content/banner/slides.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    startDate: "6/03/2019"
```
Comment (pkosiec) [inline content/banner/slides.yml]: Sorry, I've just noticed that the `startDate` is provided in incorrect way.

File: content/banner/slides.yml (hunk 1 above)
Comment (mszostok) [reply content/banner/slides.yml]: it's easy to provide it and imo it's more user friendly

Comment (pkosiec) [reply content/banner/slides.yml]: it doesn't work for me, but... keep it if you want 😛

**Pull request merged**